docs/plugins/intro.rst

```diff
@@ -40,4 +40,27 @@ The full plugins CLI is described in the :ref:`reference documentation <cli_plug
 Existing plugins
 ================
 
-Officially-supported plugins are listed on the `Overhang.IO <https://overhang.io/tutor/plugins>`__ website.
+Many plugins are available from plugin indexes. These indexes are list of plugins, similar to the `pypi <https://pypi.org>`__ or `npm <npmjs.com/>`__ indexes. By default, Tutor comes with the "main" plugin index. You can check available plugins from this index by running::
+
+    tutor plugins update
+    tutor plugins search
+
+More plugins can be downloaded from the "contrib" and "wizard" indexes::
```

Comment:
When reading this the first time (before reading the code), it wasn't clear to me that "contrib" and "wizard" are keywords that Tutor will match against indexes defined in [tutor-plugins-index](https://github.com/overhangio/tutor-plugins-index).  Maybe we could add something along the lines of:

```suggestion
More plugins can be downloaded from the "contrib" and "wizard" indexes (both of which are defined in the `official index repository <https://github.com/overhangio/tutor-plugins-index>`__, and thus automatically recognized as such by Tutor)::
```

Reply:
Good catch, I'll change the wording.